python/tvm/topi/arm_cpu/mprofile/dsp/micro_kernel/multi_channel_convolve.py

```diff
@@ -179,20 +179,26 @@ def _dual_int16_channel_convolve_impl(_tensor_h, tensor_w, channels, kernel_h, k
         extern "C"
         #endif
         int32_t {_get_func_name("int16", tensor_w, channels, kernel_h, kernel_w, suffix)}(
```

Comment:
Can we also make these changes for the `int8` version of this function in the lines above?